app/controllers/sdg_management/relations_controller.rb

```diff
@@ -39,4 +43,15 @@ def check_feature_flags
       check_feature_flag(process_name)
       raise FeatureDisabled, process_name unless Setting["sdg.process.#{process_name}"]
     end
+
+    def update_notice
+      if @record.sdg_review.present?
+        flash[:notice] = t("sdg_management.relations.update.notice",
+          relatable: relatable_class.model_name.human)
+      else
+        @record.create_sdg_review!
+        flash[:notice] = t("sdg_management.relations.update_and_review.notice",
```

Comment:
Is the `flash[:notice]` assignment necessary? :thinking: I mean, what this method returns will be used in the `notice:` param of the `redirect_to` method.

Reply:
Fixed!